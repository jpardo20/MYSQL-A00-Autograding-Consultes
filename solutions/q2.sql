-- TODO: SELECT c.nom AS nom_curs, ROUND(AVG(m.notaFinal), 2) AS mitjana FROM ... GROUP BY c.nom ORDER BY c.nom ASC;
