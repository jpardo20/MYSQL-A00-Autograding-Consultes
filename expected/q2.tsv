1r de DAM	7.75
2n de DAM	NULL
